### PR TITLE
appendices added and the css changed to layout

### PR DESCRIPTION
**Pull request by HarounAlarabi**

### CODE REVIEW

File: client/src/components/Appendices.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import "../styles/appendixes.css";
+
+function Appendixes({ questionIndex, questions }) {
+  const que = questions[questionIndex];
+
+  if (!que || !que.appendix) {
```
Comment (alinosratipour) [inline client/src/components/Appendices.jsx:7]: I think we can do this as well

```
if (!que?.appendix) {
    return null; // Or return some fallback UI if you prefer
  }

  const { type, details } = que.appendix;
```

File: client/src/components/Appendices.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  return (
+    <div>
+      <table className="table appendixTable">
+        <tbody>
+          <tr className="appendixTitle">
+            <th colSpan="4">
+              <span>{que.appendix.type}</span>
+            </th>
+          </tr>
+          {que.appendix.details.map((detail, detailIndex) => (
```
Comment (alinosratipour) [inline client/src/components/Appendices.jsx:20]: then is easier to map

details.map((detail, detailIndex) =>

Comment (alinosratipour) [inline client/src/components/Appendices.jsx:20]: then is easier to map

details.map((detail, detailIndex) =>

Comment (alinosratipour) [inline client/src/components/Appendices.jsx:20]: we can then get rid of this
`{etails.map((detail, detailIndex) => (`

File: client/src/components/Appendices.jsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    return null; // Or return some fallback UI if you prefer
+  }
+
+  return (
+    <div>
+      <table className="table appendixTable">
+        <tbody>
+          <tr className="appendixTitle">
+            <th colSpan="4">
+              <span>{que.appendix.type}</span>
```
Comment (alinosratipour) [inline client/src/components/Appendices.jsx:17]: we no longer need to say 
`<span>{que.appendix.type}</span>`

we can shorten it Like below
`<span>{type}</span>`

**Review by alinosratipour — APPROVED** [review]

File: client/src/components/Appendices.jsx (hunk 1 above)
Comment (HarounAlarabi) [reply client/src/components/Appendices.jsx]: Thanks I will change it

File: client/src/components/Appendices.jsx (hunk 2 above)
Comment (HarounAlarabi) [reply client/src/components/Appendices.jsx]: done

File: client/src/components/Appendices.jsx (hunk 3 above)
Comment (HarounAlarabi) [reply client/src/components/Appendices.jsx]: done

File: client/src/components/Appendices.jsx (hunk 2 above)
Comment (HarounAlarabi) [reply client/src/components/Appendices.jsx]: thanks I have changed it

**Pull request merged**